Vizzuality/half-earth-v3
# (fix): zoom out when AOI is a protected area

## Zoom out when AOI is a protected area

### Testing instructions

- Select a country and analyze area, go to protected areas table, click on a protected area name to go AOI protected area view and close sidebar >> Check if zoom out happened.
- Make the same clicking directly in a protected area and closing the sidebar. It should do the same.
- Check what happen if clicking in a country or region AOI and closing their sidebars >> Zoom out shouldnt happen. 

### Feature relevant tickets
Comments on: [HE-679](https://vizzuality.atlassian.net/browse/HE-679)

[HE-679]: https://vizzuality.atlassian.net/browse/HE-679?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
